wire-schema/src/test/java/com/squareup/wire/schema/SchemaTest.java

```diff
@@ -476,7 +476,7 @@
       assertThat(expected).hasMessage("multiple fields share tag 1:\n"
           + "  1. name1 (message.proto at 4:3)\n"
           + "  2. name2 (message.proto at 5:3)\n"
-          + "  for extend Message (message.proto at 3:1)");
+          + "  for message Message (message.proto at 1:1)");
```

Comment:
I initially was hesitant about this message change because it seems worse, but rather the fields point directly to their declaration which is what you'd want since they would reside in different locations.